andela/hc-leo
# #157694797 configure periodic reports

**What does this PR do?**
This PR add daily and weekly report configurations. 

**Description of Task to be completed?**
The dashboard only allowed subscribing to monthly report. This feature allows users to chose the intervals of sending periodic reports, i.e daily, weekly and monthly reports.

**How should this be manually tested?**
- Clone the repo, ```cd``` into the project and run ```./manage.py runserver```. 
- Navigate to the url provided by the development server (usually ```localhost```) using a browser.
- Create an account or log in
- Create a new check and with a suitable ```period``` and ```grace```
- Ping the url provided.
- Navigate to settings ```/accounts/profile/```
- Under `Periodic reports` check allow periodic report check box, then select an interval to send the reports.

**What are relevant pivotal tracker stories?**
[#157694797](https://www.pivotaltracker.com/story/show/157694797)
